### PR TITLE
BlueMap support

### LINKED ISSUE #53: Add BlueMap Support
I believe "EventHandler" isa already working on this....

### PR DESCRIPTION
**Pull request by ItsJuls**
Closes #53

### CODE REVIEW

File: README.md
Comment (Goosius1) [inline README.md:5]: 🚧 
- Normally I don't even mention accidental extra spaces but in this specific case .... an extra space can (weirdly) make big difference to how to the help page displays. Please revert.

File: pom.xml
Comment (Goosius1) [inline pom.xml:7]: 🚧 
- Please revert this change
- We want it to be 16

File: src/main/java/io/github/townyadvanced/townyprovinces/settings/ConfigNodes.java
Comment (Goosius1) [inline src/main/java/io/github/townyadvanced/townyprovinces/settings/ConfigNodes.java:100]: 🚧 
- Please revert to 0.9
- This is to ensure the wasteland colour "wins" more often at the border

File: src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java
Comment (Goosius1) [inline src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java:238]: 🚧 
- Remove these debug methods, they are no longer needed

File: src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java
Comment (Goosius1) [inline src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java:226]: 🚧 
This is a lot of extra blank lines. Please remove.

**Review by Goosius1 — CHANGES_REQUESTED** [review]
👏 
- This looks like really good work! Well done.
- Just a few small things to fix as per the comments
- The major thing is to fix that unexpected/unknown linkage with dynmap

**Comment by ItsJuls** [comment]
wait don't merge yet

File: src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java
Comment (Goosius1) [inline src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java:155]: 🚧 
To avoid concurrency issue, use `new HashSet(TownyProvincesDataHolder.getInstance().getProvincesSet()))"`

File: src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java
Comment (Goosius1) [inline src/main/java/io/github/townyadvanced/townyprovinces/jobs/map_display/DisplayProvincesOnBlueMapAction.java:100]: 🚧 
To avoid concurrency issues, get a copy of the set before cycling e.g.
```
new HashSet(TownyProvincesDataHolder.getInstance().getProvincesSet()))
```

File: README.md
Comment (Goosius1) [reply README.md:5]: * If you can fix this, great, if not I can quickly patch it after merging.

**Review by Goosius1 — CHANGES_REQUESTED** [review]
Looks good to me except for some conflicts.

**Pull request merged**
